backend/docs/db/schema.sql

```diff
@@ -266,6 +266,21 @@ CREATE VIEW public.external_metadata_view AS
 
 ALTER VIEW public.external_metadata_view OWNER TO postgres;
 
+--
+-- Name: files; Type: TABLE; Schema: public; Owner: postgres
+--
+
+CREATE TABLE public.files (
+    id uuid NOT NULL,
+    upload_requested_at timestamp without time zone NOT NULL,
+    uploader text NOT NULL,
+    group_id integer NOT NULL,
+    published_at timestamp without time zone
```

Comment:
What does `published_at` mean? When the user uploaded? When it was made public, i.e. released?

Reply:
When the file was made public - N.B. multiple sequence entry versions can refer to one file so there isn't an exact correspondence with sequence entry release status

Reply:
I've renamed it to released at, I hope this clarifies things as 'released' is well-ish defined concept. Feel free to unresolve if its still in need of improvement.